tests/epyccel/test_epyccel_functions.py

```diff
@@ -286,10 +286,60 @@ def g(x, y):
     f = epyccel(g)
     assert f(2,1) == g(2,1)
 
+
+def test_decorator_f15(language):
+    @types('bool', 'int8', 'int16', 'int32', 'int64')
```

Comment:
This test won't work in this branch

Reply:
(it does, but only because `language` is not passed to `epyccel`. This means that this test is run twice for fortran